### ISSUE TITLE
Inconsistent release-note line items with squash merges

### ISSUE THREAD

**Issue by dkarkar92** [google-github-actions/release-please-action#545]
### TL;DR

When squash-merging to main, the release-notes sometimes contain PR title + commits, and sometimes only the PR title (not commits). I can't understand why, and I don't see a way to control what is added. 

### Expected behavior

Either only the PR titles should be added, or PR title + all commits. 

### Observed behavior

See screenshot below. 3 PRs total. 2 have 1 commit each that was squashed. And the last one is about 9 commits that was squashed. The 9 commit PR only contains the PR title in the notes, but the single commit squashes contain the PR + the individual commit. 

![Screen Shot 2022-08-04 at 3 43 07 PM](https://user-images.githubusercontent.com/3672996/182939259-a3c9626c-8790-478f-82a1-bc2724b384a4.png)

### Action YAML

```yaml
on:
  push:
    branches:
      - main
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout source
        uses: actions/checkout@v3

      - name: Read commit-types.json from configs and output for use in changelog-types
        id: changelog-types-output
        run: |
          echo "::set-output name=COMMIT_TYPES_STR::$(
            cat configs/commit-types.json| jq -r tostring
          )"

      - uses: google-github-actions/release-please-action@v3.2.9
        with:
          release-type: node
          package-name: strategy-application
          token: ${{ secrets.RELEASE_TOKEN }}
          changelog-types: ${{ steps.changelog-types-output.outputs.COMMIT_TYPES_STR }}
```

### Additional information

Any idea on why single commit squashes show both commits in the notes, while multiple commit squashes only show the PR title? It's actually less intuitive when a single commit squash is shown twice. The PR title encapsulates what that one commit was anyway.

**Comment by bcoe** [comment]
@dkarkar92 could you share a link to the repository that is exhibiting this behavior?

**Comment by dkarkar92** [comment]
@bcoe sorry it's a private repo... but I was just looking at the git log and I noticed the squash merges that have duplicate commits do not have asterisks on the commits like they typically should. I'm wondering now, does release-please ignore commits with an asterisk?

Commit for PR #602 showed up correctly in the release notes, and it has several commits, but all but one have an asterisk. 

Commits for PR #604 and #608 were squashed, but I think the author may have removed the asterisk manually? Or did Github's squash merge option not add asterisks? 

<img width="655" alt="Screen Shot 2022-08-08 at 5 17 22 PM" src="https://user-images.githubusercontent.com/3672996/183516579-2557eb4b-d8d9-45b3-9860-b0f6a5003daf.png">

**Comment by dkarkar92** [comment]
Ah I see, Github PRs with a single commit, when using the squash + merge feature, don't show asterisks in the commit modification text box. So authors who use a valid PR title that follows conventional commits, will have 2 commits (one from the title + their actual code commit).

I'm not sure what the approach should be here. Should release-please handle this and ignore the 2nd commit? Or should we get authors to modify their messages before merge?

![Screen Shot 2022-08-08 at 5 24 34 PM](https://user-images.githubusercontent.com/3672996/183517360-b588a6bc-34d1-46b0-a3f2-b127cbf69461.png)

**Comment by bcoe** [comment]
@dkarkar92 this is intentional behavior, to allow a rollup of multiple commits into a single squashed PR (I think having this ability can be valuable for a squash workflow, we use it in our own organization).

Having said this, if this is too much of a nuisance for you, you can open a feature request [here](https://github.com/googleapis/release-please), and perhaps we could make this a config setting.
